chatez/src/backend/endpoints.ts

```diff
@@ -0,0 +1,150 @@
+import axios from 'axios';
+import { User, Message } from './types';
+
+const BASE_URL = process.env.REACT_APP_BACKEND_URL;
+
+export const getMessages = async (authToken: string) => {
+  try {
+    const response = await axios.patch(
+      `${BASE_URL}/getMessages`,
+      {},
+      {
+        headers: {
+          Authorization: authToken,
+        },
+      }
+    );
+
+    // extract only necessary data
+    const messages: Message[] = response.data.map((message: Message) => ({
+      sender: message.sender,
+      recipient: message.recipient,
+      read: message.read,
+      fileUrl: message.fileUrl,
+      message: message.message,
+      dateSent: message.dateSent,
+    }));
+
+    return messages;
+  } catch (error) {
+    console.error('Error fetching messages:', error);
+    throw error;
+  }
+};
+
+export const getFriends = async (authToken: string) => {
+  try {
+    const response = await axios.patch(
+      `${BASE_URL}/getFriends`,
+      {},
+      {
+        headers: {
+          Authorization: authToken,
+        },
+      }
+    );
+
+    // extract only necessary data
+    const friends: User[] = response.data.map((friend: User) => ({
+      email: friend.email,
+      displayName: friend.displayName,
+      photoUrl: friend.photoUrl,
+      lastActive: friend.lastActive,
+    }));
+
+    return friends;
+  } catch (error) {
+    console.error('Error fetching messages:', error);
```

Comment:
incorrect console error messages in multiple places

Reply:
sorry can you explain more

Reply:
https://github.com/ChatEz-Project/ChatEz-Frontend/pull/35/commits/adcf1e1cfa52ba2bc4fad5e806a8fd369dc889c1